requirements.txt

```diff
@@ -1,8 +1,11 @@
 Flask>=1.1.2
 Flask-Restless>=0.16.0
 Flask-SQLAlchemy>=2.0
+Flask-Migrate
 SQLAlchemy>=1.0.2
 Werkzeug>=0.16.1
 psycopg2==2.7.7
+python-dotenv
 markupsafe
-requests
\ No newline at end of file
+requests
+gspread
```

Comment:
Please add next time requested actions -> pip should be executed again :)

Reply:
sorry :)
we should update also setup.py, or remove it